ecs-v2.cfndsl.rb

```diff
@@ -261,6 +263,80 @@ def pick(dct, *keys):
       NotificationTargetARN Ref(:DrainECSHookTopic)
       RoleARN FnGetAtt(:DrainECSHookTopicRole, :Arn)
     }
+
+    asg_scaling = external_parameters.fetch(:asg_scaling, {})
+    scale_up = asg_scaling.fetch('up', {})
+    scale_down = asg_scaling.fetch('down', {})
+    asg_dimensions = [{Name: 'AutoScalingGroupName', Value: Ref('AutoScaleGroup')}]
+  
+    CloudWatch_Alarm(:ScaleUpAlarm) {
+      Condition 'IsScalingEnabled'
+      AlarmDescription FnSub(scale_up.fetch(:desc, "${EnvironmentName #{component_name} scale up alarm"))
+      MetricName scale_up.fetch(:metric_name, 'CPUReservation')
+      Namespace scale_up.fetch(:namespace, 'AWS/EC2')
+      Statistic scale_up.fetch(:statistic, 'Average')
+      Period scale_up.fetch(:cooldown, '60').to_s
+      EvaluationPeriods scale_up.fetch(:evaluation_periods, '5').to_s
+      Threshold scale_up.fetch(:threshold, '70').to_s
+      AlarmActions [Ref(:ScaleUpPolicy)]
+      ComparisonOperator scale_up.fetch(:operator, 'GreaterThanThreshold')
+      Dimensions scale_up.fetch(:dimensions, asg_dimensions)
+    }
+  
+    CloudWatch_Alarm(:ScaleDownAlarm) {
+      Condition 'IsScalingEnabled'
+      AlarmDescription FnSub(scale_down.fetch(:desc, "${EnvironmentName #{component_name} scale down alarm"))
+      MetricName scale_down.fetch(:metric_name, 'CPUReservation')
+      Namespace scale_down.fetch(:namespace, 'AWS/EC2')
```

Comment:
@Guslington Should the default metric be AWS/ECS instead of ec2?
https://github.com/theonestack/hl-component-ecs/blob/master/ecs.cfndsl.rb#L249